lib/ext/patient.rb

```diff
@@ -135,4 +140,15 @@ def randomize_demographics(patient, changed, random: Random.new)
     # HDS helpers
     #
   end
+
+  class BundlePatient < Patient; end
+
+  class VendorPatient < Patient; end
```

Comment:
I assume `VendorPatient` is associated with the BYOPatients? If so, I'm ok with waiting on adding tests for them.

Reply:
VendorPatient is associated with BYOPatients, as soon as we have the actual functionality we can add tests but at this time a VendorPatient is really just nothing more than a `_type: CQM::VendorPatient` field in the DB.

Reply:
That's what I figured. Sounds good!